src/main/java/mate/controller/CreateManufacturerController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Manufacturer;
+import mate.service.ManufacturerService;
+
+public class CreateManufacturerController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private Manufacturer manufacturer;
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/taxi/manufacturers/createManufacturer.jsp")
```

Comment:
Now as well as you have different dirs for each entity, you can name jsp files much simpler. 
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/taxi/manufacturers/create.jsp")
```
or 
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/taxi/manufacturers/add.jsp")
```